target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/ar9_netgear_dgn3500.dtsi

```diff
@@ -158,7 +158,7 @@
 
 			partition@10000 {
 				reg = <0x10000 0x10000>;
-				label = "uboot-env";
+				label = "uboot_env";
```

Comment:
In ramips we have "u-boot-env" in 235 cases. So, I'm unconvinced yet which would be the "best" name here ...

Reply:
So I thought for a long time about which is the better choice. I decided to use the underscore, because it was more common in the lantiq target. But personally, I would prefer "uboot-env". So, what's your opinion?

Reply:
Oh, I see that in ramips there is another dash. Hmm... really difficult.

Reply:
```
$ grep -R boot arch/*/boot/dts/ | grep env | grep "=" | cut -d'=' -f2 | sort | uniq -c | sort -n
      1  "bootldr-env";
      1  "bootloader environment";
      1  "console
      1 nand:512k(xloader),1920k(uboot),128k(uboot-env),10m(boot),-(rootfs) */
      1  "u_boot_env";
      1  "ubootenv";
      1  "uboot_env0";
      1  "uboot-env1";
      1  "uboot_env1";
      1  "u-boot-env2";
      1  "uboot-env2";
      1  "u-boot-env.backup1";
      1  "u-boot environment";
      1  "u-boot-environment";
      1  "uboot-environment";
      1  "uboot-envr (spi)";
      1  "uboot-env (spi)";
      2  "permanent u-boot env";
      2  "qspi: bootloader env";
      2  "qspi: bootloader env redundant";
      3  "alt-u-boot-env";
      3  "boot environment";
      3  "QSPI.u-boot-env.backup1";
      4  "uboot environment";
      5  "NAND.u-boot-env";
      5  "NAND.u-boot-env.backup1";
      5  "QSPI.u-boot-env";
      5  "uboot_env";
      6  "bootloader env redundant";
      6  "u-boot env";
      6  "uboot-env";
     11  "bootloader env";
     29  "u-boot-env";
```

so ``u-boot-env`` (and it's variations) seem to be the most popular choice upstream (based on this dumb one-liner which can be executed in the Linux kernel tree).
thus I vote for this option for consistency

Reply:
Thanks Martin! A really nice analysis. I changed it to "u-boot-env". For now. :-)